mappings/net/minecraft/block/RedstoneWireBlock.mapping

```diff
@@ -45,6 +45,11 @@ CLASS net/minecraft/unmapped/C_hbdlqish net/minecraft/block/RedstoneWireBlock
 		ARG 2 random
 		ARG 3 pos
 		ARG 4 color
+		ARG 5 support
+		ARG 6 connection
+		ARG 7 offset
+		ARG 8 randomFactor
+			COMMENT higher values make particles both more likely and further randomly offset
```

Comment:
```java
float h = randomFactor - offset;
// ...
float j = offset + h * random.nextFloat();
```
It's "more random" because it's random over a greater range, but it's also more likely to by further.
I'm not sure which wording is clearer.